routes/tasks.go

```diff
@@ -0,0 +1,111 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	m "github.com/kirikami/go_exercise_api/database/models"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := m.Task{}
+
+	if err := c.Bind(&task); err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	if err := h.DB.Save(&task).Error; err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	return c.JSON(http.StatusCreated, task)
+}
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	idParam := c.Param("id")
+
+	id, err := u.ParseIdInt64FromString(idParam)
```

Comment:
`id, err := u.ParseIdInt64FromString(c.Param("id"))` do not create local variable if you won't use it in more than two places OR you have to create local variable in order to explain what it is